gcn/notices/svom/alert.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/svom/alert.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "unevaluatedProperties": false,
+  "title": "SVOM Alert",
+  "description": "ECLAIRs or GRM trigger parameters computed onboard",
+  "allOf": [
+    { "$ref": "../core/AdditionalInfo.schema.json" },
+    { "$ref": "../core/Alert.schema.json" },
+    { "$ref": "../core/DateTime.schema.json" },
+    { "$ref": "../core/Event.schema.json" },
+    { "$ref": "../core/Localization.schema.json" },
+    { "$ref": "../core/Reporter.schema.json" },
+    { "$ref": "../core/Statistics.schema.json" }
+  ],
+  "properties": {
+    "$schema": true,
+    "notice_level": {
+      "enum": ["N1e", "N1g"],
+      "description": "SVOM notice reference id."
```

Comment:
In Core Schema, we have "id" field in Event Schema, that should serve for instrument id.

Reply:
Actually this is not just a field specifying the instrument, but the type of notice produced. In the SVOM terminology N1 notices corresponds to the first notice (trigger notice) SVOM produces.

Reply:
Ah Ok! Both are N1 (first notices), what are 'e' and 'g'? Better description is  required for users. 
Record_number field from Reporter.schema.json might be useful if multiple triggers are expected.

Reply:
Indeed "e" and "g" stand for the instrument... I will update the description